sql/item_mods.sql

```diff
@@ -27828,6 +27830,7 @@ INSERT INTO `item_mods` VALUES (15593,363,10); -- HIGH_JUMP_ENMITY_REDUCTION: 10
 INSERT INTO `item_mods` VALUES (15594,1,30);  -- DEF: 30
 INSERT INTO `item_mods` VALUES (15594,5,25);  -- MP: 25
 INSERT INTO `item_mods` VALUES (15594,357,2); -- BP_DELAY: 2
+INSERT INTO `item_mods` VALUES (15594,140,5); -- SPIRIT_CAST_REDUCTION: 5
```

Comment:
Could this modifiers be placed in order?
That is, mod 140 before mod 357

Reply:
yep we go by primary key anytime a table has one. which in this case is
```sql
  PRIMARY KEY (`itemId`,`modId`)
```

👍